__tests__/infrastructure/AccountService.spec.js

```diff
@@ -1,26 +1,175 @@
-import { AccountService } from '../../src/infrastructure'
-import networkConfig from '../config/network.conf.json'
-import { constructHttp } from '../utils'
-
-beforeAll(async() => {
-    jest.setTimeout(10000)
-    await constructHttp()
-});
+import { AccountService, ChainService, NamespaceService, NodeService } from '../../src/infrastructure';
+import TestHelper from '../TestHelper';
+import { restore, stub } from 'sinon';
+import { Mosaic, UInt64, MosaicId } from 'symbol-sdk';
 
 describe('Account Service', () => {
+	describe('getAccountInfo should', () => {
+
+		let getAccount = {};
+		let getChainInfo = {};
+		let getAccountsNames = [];
+
+		beforeEach(() => {
+			getAccount = stub(AccountService, 'getAccount');
+			getChainInfo = stub(ChainService, 'getChainInfo');
+			getAccountsNames = stub(NamespaceService, 'getAccountsNames');
+		});
```

Comment:
~~since this describe contain only one `it`, I guess you can inline?~~

ignore